adafruit_platformdetect/board.py

```diff
@@ -403,13 +403,13 @@ def _tegra_id(self):
         """Try to detect the id of aarch64 board."""
         board_value = self.detector.get_device_model()
         board = None
-        if 'tx1' in board_value:
+        if 'tx1' in board_value.lower():
             board = JETSON_TX1
-        elif 'quill' in board_value:
+        elif 'quill' in board_value or "storm" in board_value or "lightning" in board_value:

```

Comment:
While we are fixing this, we should convert the code to detect the board based on the DT compatible property rather than the DT model property. While both properties are 100% required to be an ABI, there is more likelihood that the model parameter won't actually be treated as an ABI, since it's free-form text which people are tempted to use for marketing/branding purposes, which indeed was the root-cause of the current problem that this patch fixes.

Reply:
@swarren - thanx for the info, is that something you can PR?

Reply:
@bibekbasu are you able to update the detection method too, or would you like me to look at that after this is merged? https://github.com/NVIDIA/jetson-gpio/blob/master/lib/python/Jetson/GPIO/gpio_pin_data.py#L231 shows how Jetson.GPIO detect uses the compatible property.

Reply:
I will change the detection method from model to compatible in a new patch this week